src/Microsoft.TemplateSearch.TemplateDiscovery/Microsoft.TemplateSearch.TemplateDiscovery.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <SignAssembly>false</SignAssembly>
+    <DelaySign>false</DelaySign>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\dotnet-new3\JsonDomFactory.cs" Link="JsonDomFactory.cs" />
+    <Compile Include="..\dotnet-new3\NewtonsoftJsonDocumentObjectModel.cs" Link="NewtonsoftJsonDocumentObjectModel.cs" />
+    <Compile Include="..\dotnet-new3\StreamBufferWriterImpl.cs" Link="StreamBufferWriterImpl.cs" />
+    <Compile Include="..\dotnet-new3\SystemTextJsonDocumentObjectModel.cs" Link="SystemTextJsonDocumentObjectModel.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(SrcDir)Microsoft.TemplateEngine.Cli\Microsoft.TemplateEngine.Cli.csproj" />
+    <ProjectReference Include="$(SrcDir)Microsoft.TemplateEngine.Orchestrator.RunnableProjects\Microsoft.TemplateEngine.Orchestrator.RunnableProjects.csproj" />
+
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
```

Comment:
It looks like we now get the version of this package provided for us, changing this to `<PackageReference Include="Newtonsoft.Json" />` would probably be good